### PR TITLE
update for proper Texan

### PR DESCRIPTION
**Pull request by cfont**
Texans would not say "howdy there" but instead they would say "howdy y'all"

### CODE REVIEW

**Comment by jamesob** [comment]
I owe you one of these:
shiner

Fix them tests and we'll get this baby merged!

File: examples/hello.sh [hunk 1]
```diff
@@ -7,4 +7,4 @@ hi() {
 }
 
 # Use if you're from Texas.
-alias howdy="echo howdy there"
+alias howdy="echo howdy y\'all"
```
Comment (jamesob) [inline examples/hello.sh]: no need for the backslash there since we're using double quotes on the outside

**Comment by cfont** [comment]
I removed the backslash within the web based github interface thinking this would be simple but after it automatically committed and ran through CI the tests failed. So, I cloned the repo and tested it myself and noticed that without the backslash on a Mac OSX 10.10.4 and bash 3.2.57 this little micro change does indeed fail. Once I re-add the backslash everything seems to work again. 

How would you like to proceed?

**Comment by benwr** [comment]
What happens if you make it a single-quoted string?

**Comment by benwr** [comment]
http://stackoverflow.com/questions/1250079/how-to-escape-single-quotes-within-single-quoted-strings

**Comment by cfont** [comment]
OK, so that still didn't work through the CI build. Not sure what to do but I'll keep working on it. I hate that it "works for me" because that makes it a little more difficult to troubleshoot, however, I can start up a docker image just like the CI build does and see what I can figure out.

**Comment by jamesob** [comment]
True Texan authenticity is makin' us work for it. I'm pretty sure the failure is due to tests expecting the old string.

**Comment by jamesob** [comment]
Boom. Thanks, @cfont.

**Pull request merged**